pkg/constants.go

```diff
@@ -3,4 +3,11 @@ package pkg
 const (
 	// FunctionNamespace is the default containerd namespace functions are created
 	FunctionNamespace = "openfaas-fn"
+
+	// faasdNamespace is the containerd namespace services are created
+	faasdNamespace = "openfaas"
```

Comment:
The function namespace is public, but the core services is private?  is there a good reason for the mismatch or should we make both public incase anyone wants/needs to hack or extend faasd?

Reply:
Yes I looked into the usage and the `FunctionNamespace` is required outside of the package, which is why it's public. I decided to keep the faasdNamespace private since it's not required elsewhere.